### ISSUE TITLE
Can't install 3.6.2 using Rust stable

### ISSUE THREAD

**Issue by SinisterStairs** [ijl/orjson#199]
Apologies in advance if this is considered "support for a specific environment;" I wasn't sure if that refers to user support or bug reporting.

I'm unable to compile orjson 3.6.2 with the stable version of rustc (1.54.0).  I still need to use the nightly tools, and build via maturin (0.11.2) with the  unstable-simd feature.  Versions and build log below:

- python 3.9.5
- pip 21.2.4
- maturin 0.11.2
- rustc, cargo 1.54.0 (stable)

I'm building in an Arch Linux (3.14.1) Docker container, aarch64-alpine-linux-musl.  (Host is an M1 Mac, if that matters.  FWIW, I didn't have any problems installing orjson via pip3 on the Mac itself.)

Log condensed for brevity:

```shell
$ pip3 install -v --force-reinstall orjson
  error: could not compile `orjson`
  💥 maturin failed
    Caused by: Failed to build a native library through cargo
    Caused by: Cargo build finished with "exit status: 101": `cargo rustc --message-format json --manifest-path Cargo.toml --release --lib -- -C link-arg=-s`
  🔗 Found pyo3 bindings
  🐍 Found CPython 3.9 at /usr/bin/python3
  error[E0433]: failed to resolve: use of undeclared crate or module `simdutf8`
    --> src/deserialize/deserializer.rs:32:5
     |
  32 |     simdutf8::basic::from_utf8(buf).is_ok()
     |     ^^^^^^^^ use of undeclared crate or module `simdutf8`
  Error: command ['maturin', 'pep517', 'build-wheel', '-i', '/usr/bin/python3', '--strip=True'] returned non-zero exit status 1
```
I'm able to rustup default nightly and pass the unstable-simd feature via maturin to build orjson, but it'd be nice to be able to do it via pip3 install.

**Comment by SinisterStairs** [comment]
Sorry, looks like this was addressed in PR [#198].

**Comment by ijl** [comment]
Yes, thank you for reporting. This is fixed with 3.6.3.

**Comment by SinisterStairs** [comment]
@ijl Thanks for the quick fix.  I verified orjson compiled with stable rustc in my Alpine container.
